scrapers/napa/parse.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+import csv
+import gzip
+import json
+import os
+import re
+import sys
+
+from shapely.geometry import Polygon
+
+csv.field_size_limit(sys.maxsize)
+
+ADDRESS_REGEX = re.compile('Address.*\n[^.*\>]+\>([^\<]+).*\n.*\n[^.*\>]+\>([^\<]+)')
+AMOUNTS_REGEX = re.compile('h4\>Totals\s\-.*\n.*\n.*\n.*\>Total Due\<.*\n[^\>]+\>([^\<]+)')
+GEOJSON_FILE = '/home/alin/Downloads/napa/napa.geojson'
+OUTPUT_FILE = '/home/alin/code/prop13/scrapers/napa/parse_output.csv'
```

Comment:
We probably don't want to merge this in with your paths, so here's a proposal:

* Create an `outputs` directory at the root of the repo with a README that gitignores everything else put in it
* The README states that this directory is used only for localized runs and that data should not be included
* We calculate the path for the current script using:

```
import os 
output_path = os.path.join("..", "..", "outputs", "napa")
os.mkdir(output_path)
OUTPUT_FILE = os.path.join(output_path, "parse_output.csv")
```

Reply:
Good suggestion! I've done it for this county, and also applied it to solano county which I had previously worked on